wasm/src/main/java/com/dylibso/chicory/wasm/types/CodeSection.java

```diff
@@ -1,14 +1,25 @@
 package com.dylibso.chicory.wasm.types;
 
+import java.util.ArrayList;
+import java.util.List;
+
 public class CodeSection extends Section {
-    private FunctionBody[] functionBodies;
+    private final ArrayList<FunctionBody> functionBodies;
```

Comment:
are we going to pay a runtime penalty for not using a primitive array and doing the unboxing in the accessor function?

Reply:
No, it should be about equal. It also saves us from having to grow the array to add more items (once that support is added, which should be in the next PR).